package.json

```diff
@@ -8,8 +8,8 @@
     ".": "./src/index.js"
   },
   "scripts": {
-    "format": "yarn prettier . --ignore-unknown --write",
-    "lint": "yarn prettier . --ignore-unknown --check"
+    "format": "yarn prettier . --write",
+    "lint": "yarn prettier . --check"
```

Comment:
I forgot running Prettier on dir won't glob unknown files, so `--ignore-unknow` won't be needed.